rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/NodeSource.java

```diff
@@ -400,6 +407,23 @@ private RMNode buildRMNode(Node node, Client provider) {
         return rmnode;
     }
 
+    public boolean setNodeAvailable(RMNode node) {
+        Node proactiveProgrammingNode = node.getNode();
+        String proactiveProgrammingNodeUrl = proactiveProgrammingNode.getNodeInformation().getURL();
+        Node downNode = downNodes.remove(proactiveProgrammingNodeUrl);
+
+        if (downNode != null) {
+            logger.info("Setting node as available: " + proactiveProgrammingNodeUrl);
```

Comment:
Probably many but to understand network issues in production it is really useful. Logs can be filtered. It is always better to have more information than no information. If multiple people ask for reducing the verbosity I will do but otherwise, I prefer to keep as it is.